common/locales/en.json

```diff
@@ -91,6 +91,7 @@
         "toggleAsbplayerSubtitleTracks": "Toggle subtitle track in asbplayer",
         "toggleAutoPause": "Toggle auto-pause",
         "toggleCondensedPlayback": "Toggle condensed playback",
+        "toggleFastForwardPlayback": "Toggle fast forward playback",
```

Comment:
If you run `node merge-loc-from-en.js` inside `scripts` then you can copy this line to the loc files for all of the other languages
With the way asbplayer works right now, all of the keys need to exist in all of the loc files.